managed/services/config/pmm-managed.yaml

```diff
@@ -20,6 +21,7 @@ services:
           password: pmm
     reporting:
       send_on_start: false
+      send_on_start_env: "PERCONA_TEST_TELEMETRY_DISABLE_START_DELAY"
```

Comment:
Why do we need to define this in config? All ENV names are barcoded and I see no case when it can and really will be redefined by user